editstyle.css

```diff
@@ -0,0 +1,111 @@
+*{
+    font-family: Arial, sans-serif;
+    margin: 0;
+    padding: 0;
+}
+.container {
+    max-width: 1300px;
+    margin: 0 auto;
+    padding: 20px;
+    display: flex;
+    flex-direction: column;
+    min-height: 100vh;
+}
+.header {
+    background-color: #333;
+    padding: 15px;
+    text-align: end;
+}
+.header a {
+    color: #fff;
+    text-decoration: none;
+    font-size: 20px;
+}
+.main {
+    display: flex;
+    flex: 1;
+    margin-top: 20px;
+}
+.about{
+    background-color: aquamarine;
+}
+.content {
+    flex: 1;
+    padding: 1rem;
+    margin-right: 20px;
+}
+.sidebar {
+    background-color: #f4f4f4;
+    padding: 1em;
+    width: 25%;
+    max-width: 300px;
+    text-align: center;
+}
+.sidebar img {
+    width: 100px;
+    height: 100px;
+    border-radius: 50%;
+    margin-bottom: 1em;
+}
+.sidebar h2, .sidebar h3 {
+    text-align: center;
+}
+.footer {
+    background-color: #333;
+    color: #fff;
+    text-align: center;
+    padding: 1em;
+    margin-top: 20px;
+}
+.experience {
+    max-height: 300px;
+    overflow-y: auto;
+    margin-bottom: 20px;
+}
+.experience-form {
+    display: none;
+    margin-bottom: 10px;
+}
+.experience-entry {
+    border: 1px solid #ddd;
+    padding: 10px;
+    margin-bottom: 10px;
+}
+.form-field {
+    display: flex;
+    margin-bottom: 20px;
+    align-items: center;
+}
+.form-field label {
+    width: 80px;
+    margin-left: 20px;
```

Comment:
try to manage through gap instead of margin

Reply:
done